### PR TITLE
made new auton for getting 2nd note from midline needs testing

### PR DESCRIPTION
**Pull request by Griffin9881**
### Pull Request

---------------------------------------------

#### Issue Number

Closes #123

### CODE REVIEW

**Comment by reediculous456** [comment]
I think that moving forward we should test all code before merging it to main. Now that we have the robot we can say that all code on main is tested/working if we do that

**Comment by Griffin9881** [comment]
what are you talking about, i dont disagree im just confused where this is coming from

File: src/main/java/frc/robot/autons/ScoreInAmpMidline.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+                new ScoringElevatorPositionCommand(ElevatorSetpoints.Amp, FinishActions.NoDisable),
+                new ScoreNoteCommand(ScoringLocation.Amp, 1.5),
+                new ScoringElevatorPositionCommand(ElevatorSetpoints.Ground),
+                new ParallelCommandGroup(
+                        followPathCommand(Paths.pathFromAmpToMidlineWall)
+                                .until(acquisition::isNotePresent)
+                                .andThen(followPathCommand(Paths.pathFromMidlineToAmp)),
+                        // new FullAcquireCommand()),
+                        new AcquisitionRunCommand()),
+                new ScoringElevatorPositionCommand(ElevatorSetpoints.Amp, FinishActions.NoDisable),
```
Comment (matunkar) [inline src/main/java/frc/robot/autons/ScoreInAmpMidline.java:38]: i feel like this can be done while following the pathfrommidlinetoamp

Comment (Griffin9881) [reply src/main/java/frc/robot/autons/ScoreInAmpMidline.java:38]: we want to make sure that the note gets all the way to top of storage as sometimes it doesnt go up right away

**Pull request closed without merging**